subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/JavaToolchainUpToDateIntegrationTest.groovy

```diff
@@ -41,7 +41,9 @@ class JavaToolchainUpToDateIntegrationTest extends AbstractPluginIntegrationTest
     }
 
     def "compile and test not up-to-date once toolchain changed"() {
-        def someJdk = AvailableJavaHomes.getDifferentJdk()
+        def someJdk = AvailableJavaHomes.getDifferentVersion()
+        println "someJdk = $someJdk"
+        println "Jvm.current() = ${Jvm.current()}"
```

Comment:
Is this for debugging purpose? Shall we remove them?

Reply:
Oops, thanks for catch